.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
```

Comment:
Oh, now I got it. No, it's not visual only.

What happens is that when we are using `indent_style = tab` with `indent_size = 2` we are configuring our tab to use 2 spaces by default. So, in this case, every tab will be based in 2 spaces.

We idea is define the configuration in the repository and every collaborator should follow this configuration to avoid files with different styles/code standards. If someone tries to unfollow this default configuration, the build will breaks.